test/common/buffer/utility.h

```diff
@@ -0,0 +1,38 @@
+#pragma once
+
+#include <initializer_list>
+
+#include "common/buffer/buffer_impl.h"
+
+#include "gtest/gtest.h"
+
+namespace Envoy {
+namespace Buffer {
+namespace {
+
+inline void addRepeated(Buffer::Instance& buffer, int n, int8_t value) {
+  for (int i = 0; i < n; i++) {
+    buffer.add(&value, 1);
+  }
+}
+
+inline void addSeq(Buffer::Instance& buffer, const std::initializer_list<uint8_t> values) {
+  for (int8_t value : values) {
+    buffer.add(&value, 1);
+  }
+}
+
+inline void addString(Buffer::Instance& buffer, const std::string& s) { buffer.add(s); }
```

Comment:
Let's not move this one into common: people should just call buffer.add(s). If you want to leave it in the thrift code I'll remove it later. 

I added it for conformity with the other utility methods, but because many of them are now just methods on Buffer there's no reason to provide this.

Reply:
removed and updated references